tdrs-backend/tdpservice/parsers/schema_defs/tanf/t4.py

```diff
@@ -0,0 +1,48 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import SchemaManager
+from ...fields import Field
+from ...row_schema import RowSchema
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T4
+
+
+t4 = SchemaManager(
+      schemas=[
+        RowSchema(
+          model=TANF_T4,
+          preparsing_validators=[
+              validators.hasLength(71),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item=1, name='RecordType', type='string', startIndex=0, endIndex=2,
```

Comment:
`item` should be a string following #2384

Reply:
Yes they should be. Im noticing not all defs use the string though. I.E. m1/m2 and t1/t2. Not sure why that happened. I can just update them all as apart of this.